docs/tutorials/FlowTutorial.md

```diff
@@ -0,0 +1,1630 @@
+
+```
+worst slack -1.35
+tns -95.25
+wns -1.35
+```
+
+To learn more about visualizing and tracing time paths across the design
+hierarchy refer to the OpenROAD [GUI](../../src/gui/README.md)
+
+### Power
+
+Now use the report command to view individual power components i.e.
+sequential, combinational, macro and power consumed by I/O pads.
+
+```
+report_power
+```
+
+The power output as follows:
+
+```
+finish report_power
+--------------------------------------------------------------------------
+Group                  Internal  Switching    Leakage      Total
+                          Power      Power      Power      Power
+----------------------------------------------------------------
+Sequential             5.58e-03   6.12e-04   1.67e-08   6.19e-03  19.0%
+Combinational          9.23e-03   1.71e-02   4.90e-08   2.63e-02  81.0%
+Macro                  0.00e+00   0.00e+00   0.00e+00   0.00e+00   0.0%
+Pad                    0.00e+00   0.00e+00   0.00e+00   0.00e+00   0.0%
+----------------------------------------------------------------
+Total                  1.48e-02   1.77e-02   6.57e-08   3.25e-02 100.0%
+                          45.6%      54.4%       0.0%
+```
+
+## OpenROAD GUI
+
+The GUI allows user to select, control, highlight and navigate across the
+design hierarchy and design objects (nets, pins, instances, paths etc.)
+through detailed visualization and customization options. Find details
+on how to use the GUI here:
+
+```
+less ../../src/gui/README.md
+```
+
+In this section learn how to:
+
+1.  Visualize design hierarchy
+2.  Load DEF and LEF files for floorplan and layout visualization
+3.  Trace the synthesized clock tree to view hierarchy and buffers
+4.  Use heat maps to view congestion and observe the effect of placement
+5.  View and trace critical timing paths
+6.  Set display control options
+
+If user have completed the RTL-GDS flow, then proceed to view the final
+GDS file under results directory `./results/sky130hd/ibex/base/`
+
+For the `ibex` `DESIGN_CONFIG` uncomment the `DESIGN_CONFIG` variable
+in the `config.mk`:
+
+```
+# DESIGN_CONFIG=./designs/sky130hd/gcd/config.mk
+DESIGN_CONFIG=./designs/sky130hd/ibex/config.mk
+# DESIGN_CONFIG=./designs/sky130hd/aes/config.mk
+```
+
+```
+make gui_final
+```
+
+### Viewing Layout Results
+
+The `make gui_final` command target successively reads and loads the
+technology `.lef`, design `.def` files and the parasitics and invokes the
+GUI in these steps:
+
+-   Reads and loads `.lef` files (tech and merged)
+-   Reads and loads `.def` files (final design def)
+-   Loads `.spef` (parasitics)
+
+The figure below shows the post-routed DEF for the `ibex` design.
+
+![ibex_fina_def](./images/ibex_final_def.png)
+
+### Visualizing Design Objects And Connectivity
+
+Note the `Display Control` window on the LHS (C- Color, V- Visible, S-
+Selectable) for various design objects: Layers, Nets, Instances,
+Blockages etc.

```

Comment:
C, V, S are no longer in the GUI, need to update the screenshot.

Reply:
@arlpetergadfort in OpenROAD GUI LHS those icons still exist. But abbreviation not there.
![cvs](https://user-images.githubusercontent.com/85787913/150082016-a345f112-d301-4da5-8821-1a10cf9e731a.png)

Reply:
@vijayank88 , Please update the screenshot and resolve